dao-contracts/src/rules/mod.rs

```diff
@@ -1,22 +1,40 @@
 use casper_dao_utils::casper_env::revert;
 
-use crate::rules::validation::Validation;
+use crate::{
+    rules::validation::{Validation, VotingValidation},
+    voting::voting_state_machine::VotingStateMachine,
+};
 
 pub mod action;
 pub mod builder;
 pub mod validation;
 
 pub struct Rules {
     pub validations: Vec<Box<dyn Validation>>,
+    pub voting_validations: Vec<Box<dyn VotingValidation>>,
 }
 
 impl Rules {
-    pub fn validate(&self) {
+    pub fn validate_validations(&self) {
```

Comment:
`validate_validations()` :thinking: sounds weird. Maybe we should validate rules, not validations.